activesupport/CHANGELOG.md

```diff
@@ -1,3 +1,66 @@
+*   Introduce `ActiveSupport::ActionableError.define`.
+
+    This can be useful if you're code does not depend strictly on Rails. In
```

Comment:
`your`. Should this say Rails or just Active Support? It's a weird boundary that we mention Rails in the second line even though this is an Active Support feature. So maybe we should mention how to catch and triggers these in people's own apps?

Reply:
Riiiight, we're trying to allude to that this depends on the Action Pack raise/rescue to get built in support.